wolfssl/wolfcrypt/types.h

```diff
@@ -723,6 +723,7 @@ typedef struct w64wrapper {
             #define XMEMMOVE(d,s,l)   memmove((d),(s),(l))
 
         #define XSTRLEN(s1)       strlen((s1))
+        #define XSTRCPY(s1,s2)    strcpy((s1),(s2))
```

Comment:
Extra parentheses in macros are always a good idea to avoid unexpected errors when the macro is expanded. Either way, `strcpy` is not actually needed anymore as I've been able to satisfy the static analyzer with `memcpy` instead.